example/index.js

```diff
@@ -1,41 +1,34 @@
-// http app server
-import Koa from 'koa';
+import fs from 'fs';
+import path from 'path';
 
 // library
 import { createElement, createRenderer } from '../dst';
 
 // PDF to render
 import PDF from './components/root';
```

Comment:
Might be worth renaming this to something less generic while we're in here - `InvoicePDF`?

Reply:
I thought about that but left it like this to remove extra cognitive load when comprehending the example; I don't feel strongly though.

Reply:
Yep - nbd either way.  Already approved and more than happy for this to be merged as-is :)